include/BNO055.hpp

```diff
@@ -63,6 +64,17 @@ namespace IMU {
  */
 class BNO055 {
 public:
+    /**
+     * Represents potential errors that may take place when using the I2C
+     * interface. Each method that interfaces over I2C could potentially
+     * return one of these errors, or OK if no error.
+     */
+    enum class BNO055Status {
+        FAIL_INIT = 0,
+        FAIL_SELF_TEST = 1,
+        SUCCESS = 2
```

Comment:
This is a nitpick, but generally, we have the successful state have a value of 0, and we call it "OK." It's not really an official standard, but it's pretty common across C code I've seen in my experience